rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -20,12 +20,16 @@
 
 # Bootstrap script for docker container.
 
+touch "/alive/docker_$PAI_CONTAINER_ID"
```

Comment:
>/alive/docker_$PAI_CONTAINER_ID [](start = 7, length = 31)

you mean /alive/$APP_ID/docker_$CONTAINER_ID?

Reply:
no, `/tmp/pai-root/alive/$APP_ID` in yarn container is mounted as `/alive` in docker container.